puppet/modules/aws_ebs/functions/disks.pp

```diff
@@ -0,0 +1,7 @@
+function aws_ebs::disks() >> Array[String] {
+  if $::disks == undef {
+    []
+  } else {
+    $::disks.keys.sort
```

Comment:
This will fail for more than 10 nvme disks

```
echo nvme0n1 nvme1n1 nvme2n1 [...] nvme10n1 nvme11n1 | xargs -n 1 | sort
nvme0n1
nvme10n1
nvme11n1
nvme1n1
nvme2n1
[...]
```

This should be the correct order:

```
nvme0n1
nvme1n1
nvme2n1
[...]
nvme10n1
nvme11n1
```

We should address and test this